src/Adapter/Driver/Pdo/Result.php

```diff
@@ -107,7 +107,7 @@ public function isBuffered()
      */
     public function setFetchMode($fetchMode)
     {
-        if ($fetchMode < 1 || $fetchMode > 10) {
+        if (($fetchMode < 1 || $fetchMode > 12) && $fetchMode !== \PDO::FETCH_CLASS | \PDO::FETCH_PROPS_LATE) {
```

Comment:
I don't think these should ever be supported: they are genuinely terrible if not actively harmful

Reply:
@Ocramius Could you elaborate? FETCHL_NAMED solved my problem without a hitch. Am I totally missing something obvious maybe?

Reply:
@Ocramius friendly reminder via notification ;)